source/geometry_model/spherical_shell.cc

```diff
@@ -337,6 +354,15 @@ namespace aspect
       return std::map<std::string,types::boundary_id>();
     }
 
+    template <int dim>
+    std::set< std::pair< std::pair<types::boundary_id, types::boundary_id>, unsigned int> >
+    SphericalShell<dim>::
+    get_periodic_boundary_pairs () const
+    {
+      std::set< std::pair< std::pair<types::boundary_id, types::boundary_id>, unsigned int> > periodic_boundaries;
+      periodic_boundaries.insert( std::make_pair( std::pair<types::boundary_id, types::boundary_id>(2, 3), 1) );
```

Comment:
this "1" needs to  be checked. It is either 0 or one, not sure.

Reply:
changed them all to 0 for now